2017/Team-Training/OMERO-web/index.html

```diff
@@ -0,0 +1,485 @@
+        </script>
+
+        <!--[if lt IE 9]>
+        <script src="lib/js/html5shiv.js"></script>
+        <![endif]-->
+    </head>
+
+    <body class="ome_theme">
+
+        <div class="reveal">
+
+            <!-- Any section element inside of this container is displayed as a slide -->
+            <div class="slides">
+
+                <!-- title page -->
+                <section>
+                    <h1>Getting started with OMERO.web</h1>
+                    <p>Will Moore</p>
+                </section>
+                
+                <section>
+                    <h2>Outline</h2>
+                    <ul>
+                        <li>OMERO.web and it's apps</li>
+                        <li>Relationship between apps</li>
+                        <li>Browser - Server communication</li>
+                        <li>OMERO.web JavaScript (jQuery)
+                          <ul>
+                            <li>Where is the OMERO.web JavaScript code?</li>
+                            <li>jQuery intro and Browser devtools</li>
+                            <li>Selectors</li>
+                            <li>DOM manipulation</li>
+                            <li>AJAX</li>
+                            <li>Other libraries we use</li>
+                            <li>jsTree</li>
+                          </ul>
+                        </li>
+                    </ul>
+                </section>
+
+
+                <section>
+                    <h2>OMERO.web framework</h2>
+
+                    <img style="height:500px" src="https://www.openmicroscopy.org/site/support/omero5.3/_images/OmeroWeb.png"/>
+                    <br>
+                    <a href="https://www.openmicroscopy.org/site/support/omero5/developers/Web.html">OMERO.web documentation</a>
+                </section>
+
+
+                <section>
+                    <h2>Relationship between apps</h2>
+
+                    <ul>
+                        <li>webgateway, webclient, webadmin, feedback, api in
+                            <a href="https://github.com/openmicroscopy/openmicroscopy/tree/develop/components/tools/OmeroWeb/omeroweb">components/tools/OmeroWeb/omeroweb</a>.
+                        </li>
+                        <li>
+                            Configured in <a href="https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroWeb/omeroweb/settings.py">OmeroWeb/omeroweb/settings.py</a>
+                        </li>
+                        <li>OMERO.web core apps never installed independently.</li>
+                        <li>Django apps <i>CAN</i> link to or import from other installed apps</li>
+                        <li>
+                            webgateway provides <a href="http://www.openmicroscopy.org/site/support/omero5/developers/Web/WebGateway.html">core functionality</a>: 
```

Comment:
Perhaps a mention of API stability/public-ness for each different core section (here or elsewhere)